geonode_mapstore_client/client/js/components/Permissions/GeoLimits.jsx

```diff
@@ -174,4 +184,68 @@ GeoLimits.defaultProps = {
     mapType: 'openlayers'
 };
 
-export default GeoLimits;
+const ConnectedGeoLimits = connect(
+    createSelector(
+        [getResourceId, mapSelector, layersSelector],
+        (resourceId, config, layers) => ({
+            resourceId: resourceId || config?.info?.id,
+            layers: layers ?? config?.layers ?? []
+        })
+    )
+)(({ entry, onUpdate, resourceId, layers }) => {
```

Comment:
We should not connect components inside the components folder, please move this connect in the plugin

Reply:
Yeah was supposed to move it along with permissions